code/src/components/SecondQuestion.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+// This where user gets to check in a radiobutton where there wanna go
+
+const SecondQuestion = ({ onSetDestination, onStepChange }) => {
+  const travelDestion = [
+    'The Alps sking',
+    'Bahamas on the beach',
+    'New York for shoping',
+    'Vancover for fishing',
+    'The Moon',
+  ]
+  return (
+    <>
+      <section className="destination-container">
+        <h2 className="section-heading">Where do u wanna go?</h2>
+        {travelDestion.map((destination) => (
+          <label className="radiobutton" key={destination}>
+            <input
+              type="radio"
+              name="enjoy"
+              value={destination}
+              onChange={onSetDestination}
```

Comment:
Good solution to use the map-function, saves up code (I had another solution which resulted in much more rows...).